libraries/chain/steem_evaluator.cpp

```diff
@@ -2076,6 +2076,33 @@ namespace golos { namespace chain {
             _db.modify(account, [&](account_object& a) {
                 a.last_account_recovery = now;
             });
+
+            reset_vesting_withdraw(account);
```

Comment:
Checking on HF19 should be added here.

Reply:
afalaleev is right 
@e-schepachev If need example for HF constant - use this code: https://github.com/GolosChain/golos/pull/942